src/App.js

```diff
@@ -4,94 +4,182 @@ import styled from 'styled-components';
 import { confluenceLinks } from './constants';
 
 const StyledApp = styled.div`
-  text-align: center;
-`;
-
-const StyledParagraph = styled.p`
-  color: red;
+  text-align: left;
 `;
 
 const StyledHeader = styled.header`
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
+  color: black;
+  background-color: #fff;
+  min-height: 10vh;
+  margin-left: 20px;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 `;
 
-const StyledLink = styled.a`
-  color: #61dafb;
+const StyledTitle = styled.p`
+  font-size: 28px;
 `;
 
-const random = number => {
-  return Math.floor(Math.random() * (number + 1));
-};
-
-const backgroundChange = () => {
-  return `rgb(${random(255)}, ${random(255)}, ${random(255)})`;
-};
+const StyledSubTitle = styled.p`
+  font-size: 12px;
+  color: #5e6c84;
+`;
 
-const changeStyling = () => {
-  console.log('The button has been clicked and the width and font family has been changed');
-  document.body.style.width = '30%';
-  document.body.style.backgroundColor = backgroundChange();
-  document.body.style.fontFamily = 'monospace';
-};
+const StyledGrid = styled.div`
+  max-width: 896px;
+  border: 1px solid #c1c7d0;
+`;
 
-const keyPressFunction = e => {
-  switch (e.key) {
-    case 'Enter':
-      console.log('you hit enter');
-      break;
-    case 'd':
-      console.log('you hit d');
-      break;
-    default:
-      console.log('you hit something else');
-  }
-};
+const StyledTitleRow = styled.div`
+  display: flex;
+  color: rgb(0, 0, 255);
+  background-color: #e6fcff;
+  text-align: center;
+  height: 35px;
+  font-weight: bold;
+`;
 
-const App = () => {
-  const [keyPress, setKeyPress] = useState('');
+const StyledPictureRow = styled.div`
+  display: flex;
+  text-align: center;
+  height: 146px;
+`;
 
-  const outputPrinter = e => {
-    keyPressFunction(e);
-    setKeyPress(`you pressed: ${e.key}`);
-  };
+const StyledCol = styled.div`
+  flex: ${props => props.size};
+  border: 1px solid #c1c7d0;
+  padding: 7px;
+`;
 
-  const handleSubmit = e => {
-    e.preventDefault();
-    console.log(`the event is: ${e.type} and this was triggered on:`, e.target);
-  };
+const StyledImage = styled.img`
+  width: 100px;
+  height: 100px;
+  margin: 23px;
+`;
 
-  const confluenceDocs = () =>
-    confluenceLinks.map((confluenceEl, i) => (
-      <StyledLink key={i} href={confluenceEl.link}>
-        {confluenceEl.name}
-      </StyledLink>
-    ));
+const StyledLink = styled.a``;
```

Comment:
This styled component isn't necessary as there's no styles in it. We can just update the `StyledLink` to use an `<a>` tag :)